### ISSUE TITLE
Deeplab V3 TFLite starter model has wrong input details shape dtype

### ISSUE THREAD

**Issue by mayorquinmachines** [tensorflow/tensorflow#32224]
<em>Please make sure that this is a bug. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:bug_template</em>

**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): No
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Linux Ubuntu 18.04
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device: N/A
- TensorFlow installed from (source or binary): source
- TensorFlow version (use command below): 1.14.0
- Python version: 3.6
- Bazel version (if compiling from source):
- GCC/Compiler version (if compiling from source):
- CUDA/cuDNN version: N/A
- GPU model and memory: N/A

Current Behavior:
When running the starter model found [here](https://storage.googleapis.com/download.tensorflow.org/models/tflite/gpu/deeplabv3_257_mv_gpu.tflite) and printing out the input details, the dtype in the shape is labeled as "int32" but the model expects "float32"
input details = [{'name': 'sub_7',
  'index': 183,
  'shape': array([  1, 257, 257,   3], dtype=int32),
  'dtype': numpy.float32,
  'quantization': (0.0, 0)}]

When feeding an image of dtype int32:
ValueError: Cannot set tensor: Got tensor of type INT32 but expected type FLOAT32 for input 183, name: sub_7 

TLDR: dtype and shape dtype are inconsistent. The dtype in the input details shape should be float32.

**Comment by freedomtan** [comment]
The int32 dtype (`'shape': array([ 1, 257, 257, 3], dtype=int32)`) you referred to is data type for 'shape', not for the data type of the input tensor. And you can see `'dtype': numpy.float32`

**Comment by mayorquinmachines** [comment]
@freedomtan My bad! Got confused, thanks for the explanation :+1:
